Makefile

```diff
@@ -1,11 +1,12 @@
 # Config
-server=preprod@185.34.32.17
+server=${env}@185.34.32.17
 src_dir=~/cafebabel.com
 venv_dir=~/cafebabel.com/venv
 branch=master
-LOGS=access
 old_prod=prod@91.194.60.65
 old_prod_media_dir=~/cafebabel/data/medias
+prod_server=127.0.0.1:5001
```

Comment:
I guess this was for local tests?

Reply:
No, this is the server running in production with gunicorn

Reply:
Oh I just noticed the `${${env}_server}` 😈